### ISSUE TITLE
branch: master | celadon quest - Silph Scope

### ISSUE THREAD

**Issue by prouser1234** [M1L4/QuestingDebug#9]
so after giving the officer the eevee its not going to battle Giovanni or taking the Silph Scope!

and sometimes its stuck with that loop :

[16:36:27] Acces Granted!
[16:36:27] There is nothing stored in this receptor.
[16:36:28] Introduce password.

solution:
need to go to take sliph scope alone and then it is ok.

**Comment by M1L4** [comment]
@prouser1234: Did you perhaps do some of the steps manually? 
1. As far as I can tell, all my bots battled giovanni before starting to retrieve eevee. So - "``after giving the officer the eevee its not going to battle Giovanni or taking the Silph Scope!``" isn't any abnormality. 
    - Did you get any error message?
    - Did the bot continue to progress?
2. Same questions regarding Eevee loop: 
    - Did you get any error message?
    - Did the bot continue to progress?


    It is a bug, if the bot couldn't progress or it didn't receive eevee. Apart from that, since Eevee is located in a random receptor, multiple locations have to be checked. So multiple prints of the following aren't uncommon:
> [16:36:27] Acces Granted!
> [16:36:27] There is nothing stored in this receptor.
> [16:36:28] Introduce password.

Could you pls verify those points? As I had never encountered any issues here, it seems a little strange to happen now. (I don't believe they updated any part of this quest).

@All: Could someone confirm they also have an issue here? My bots as well as those of other testers', never encountered this behaviour before. If confirmed, any clues on what have changed? sliph scopes position?
Also take note, if this issue has to be split into two separate issue reports - _see my comment to prouser1234_

**Comment by M1L4** [comment]
I couldn't reproduce this error, created 5 accounts which all worked until earth badge badge without any problems. You still experiencing this issue?
Is it maybe related to lag? Did you try switching proxies?
